src/lang/english.txt

```diff
@@ -1991,6 +1991,9 @@ STR_FACE_TIE                                                    :Tie:
 STR_FACE_EARRING                                                :Earring:
 STR_FACE_TIE_EARRING_TOOLTIP                                    :{BLACK}Change tie or earring
 
+STR_NETWORK_SERVER_VISIBILITY_PRIVATE                           :Private
+STR_NETWORK_SERVER_VISIBILITY_PUBLIC                            :Public
```

Comment:
I get that this is preparing for STUN stuff, but at the moment it's just Internet/LAN, why not just reuse those strings (for now) ?

Reply:
Unless you actually meant to remove those strings too ;)

Reply:
It was called "advertised" vs "non-advertised", which is just a strange wording for this setting. It is what it technically does, but not something most people will really understand.

It has nothing to do with Internet/LAN btw. You can have a server non-advertised which people can reach via direct IP just fine.

The reason I renamed it into Private / Public, is because most games name it like that. And sure, another PR might add "friends-only" to the list, but as far as this PR is concerned, that is not important.

So Internal/LAN will always stay. That is just a completely different segment :) And my suggestion here is to use "private" / "public" for "listed when you click 'search internet'".

You agree?